src/ripple/protocol/impl/STTx.cpp

```diff
@@ -186,7 +187,7 @@ std::pair<bool, std::string> STTx::checkSign() const
         // at the SigningPubKey.  It it's empty we must be
         // multi-signing.  Otherwise we're single-signing.
         Blob const& signingPubKey = getFieldVL (sfSigningPubKey);
-        ret = signingPubKey.empty () ? checkMultiSign () : checkSingleSign ();
+        ret = signingPubKey.empty () ? checkMultiSign (rules) : checkSingleSign (rules);
```

Comment:
I think that for these two **`private`** member functions, it's a better idea to pass an actual `bool` instead of the actual `Rules` object.

Reply:
Done.